### PR TITLE
set up project dependencies, simple grunt

### PR DESCRIPTION
**Pull request by hippogriffic**
just adding bower, npm, grunt etc

most of this was copied from incuna-transitions

@incuna/frontend please review

### CODE REVIEW

File: styles/main.css [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+/*# sourceMappingURL=main.css.map */
```
Comment (hippogriffic) [inline styles/main.css:2]: does this get committed or should it be in git ignore?

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  },
+  "license": "MIT",
+  "devDependencies": {
+    "grunt": "~0.4.1",
+    "grunt-contrib-connect": "~0.5.0",
+    "grunt-contrib-cssmin": "~0.7.0",
+    "grunt-contrib-watch": "~0.5.3",
+    "grunt-sass": "1.1.0",
+    "grunt-swig-templates": "^0.1.4",
+    "grunt-version": "^0.3.0",
```
Comment (henrahmagix) [inline package.json]: Pint exact versions here. `npm list --depth 0` will give you the versions you have installed, which may not be what's here minus `^` or `~`

File: package.json [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    "jit-grunt": "0.9.1",
+    "node-bourbon": "4.2.3"
+  },
+  "dependencies": {
+    "load-grunt-tasks": "~0.2.0"
```
Comment (henrahmagix) [inline package.json]: This is a dev dependency

**Comment by hippogriffic** [comment]
@henrahmagix changes made

**Pull request merged**